fastlane/lib/fastlane/actions/update_fastlane.rb

```diff
@@ -134,21 +98,17 @@ def self.details
 
       def self.available_options
         [
-          FastlaneCore::ConfigItem.new(key: :nightly,
-                                       env_name: "FL_UPDATE_FASTLANE_NIGHTLY",
-                                       description: "Opt-in to install and use nightly fastlane builds",
-                                       is_string: false,
-                                       default_value: false),
           FastlaneCore::ConfigItem.new(key: :no_update,
                                        env_name: "FL_NO_UPDATE",
                                        description: "Don't update during this run. This is used internally",
                                        is_string: false,
                                        default_value: false),
-          FastlaneCore::ConfigItem.new(key: :tools,
-                                       env_name: "FL_TOOLS_TO_UPDATE",
-                                       description: "Comma separated list of fastlane tools to update (e.g. `fastlane,deliver,sigh`)",
-                                       deprecated: true,
-                                       optional: true)
```

Comment:
Was this intentionally removed in this PR?

Reply:
Yup yup! It is sooooo old. This is when _fastlane_ had multiple gems (one for each tool). I was triggered when I saw it so I wanted it gone too while I was in here 🙃

Reply:
Bye bye then